Build.proj

```diff
@@ -12,6 +12,7 @@
     <ProjectToBuild Include="$(MSBuildThisFileDirectory)src\*\src\*.*proj" />
 
     <!-- Build all Test Projects -->
+    <TestExclusions Condition="'$(OS)' != 'Windows_NT'" Include="$(MSBuildThisFileDirectory)src\Microsoft.Bcl.HashCode\tests\Microsoft.Bcl.HashCode.Tests.csproj" /> <!-- This test is .NET Framework only -->
     <TestExclusions Condition="'$(OS)' != 'Windows_NT'" Include="$(MSBuildThisFileDirectory)src\System.Buffers\tests\System.Buffers.Tests.csproj" /> <!-- This test is .NET Framework only -->
```

Comment:
Why don't we build and run .NET Framework tests?

Reply:
It looks to me like this condition was in place because of this: https://github.com/dotnet/maintenance-packages/blob/62823150914410d43a3fd9de246d882f2a21d5ef/src/System.Buffers/tests/System.Buffers.Tests.csproj#L4

The package doesn't contain a .NETCore implementation.  The package is expected to be dropped on .NETCore due to the inbox assembly.  Perhaps System.Buffers could try to test the framework assembly, or test that the package is dropped (though keep in mind ProjectReferences don't get dropped by conflict resolution in the same way).

That's different than Microsoft.BCL.Hashcode - it has a core implementation - even though its a facade.  We should run the test on Core too.